salt/modules/file.py

```diff
@@ -3707,10 +3707,21 @@ def is_link(path):
     return os.path.islink(os.path.expanduser(path))
 
 
-def symlink(src, path):
+def symlink(src, path, force=False):
```

Comment:
`force` is a new feature. Can you add a changelog for the new force argument and a versionadded for the argument in the module doc.